.circleci/build_lib.sh

```diff
@@ -101,7 +101,7 @@ fi
 
 }
 
-export JAVA_HOME="/usr/lib/jvm/java-8-openjdk"
+export JAVA_HOME="/usr/lib/jvm/java-11-openjdk-amd64"
```

Comment:
That `amd64` brings concerns. Why are we now platform-dependent?

Reply:
Don't know what to do here, this is the path the package (apt-get install openjdk-11-jdk) is installed... What would you recommend ?